lsdcGui.py

```diff
@@ -220,10 +220,17 @@ def __init__(self, parent = None, **kwargs):
           self.beamCheckOnCheckBox.setChecked(False)
         self.beamCheckOnCheckBox.stateChanged.connect(self.beamCheckOnCheckCB)
 
+        self.gripperUnmountColdCheckBox = QCheckBox("Unmount Cold")
+        self.gripperUnmountColdCheckBox.setChecked(False)
+        self.gripperUnmountColdCheckBox.setEnabled(False)
```

Comment:
so unmount cold is only available with queue collect, and its default is true for whenever queue collect is on?

Reply:
The default should be false when queue collect is on.